MyOwnORM/MyOwnORM/Program.cs

```diff
@@ -0,0 +1,199 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MyOwnORM
+{
+    public class Context : CustomDbContext
+    {
+        public Context(string connectionString) : base(connectionString)
+        {
+            People = new CustomDbSet<Person>(connectionString);
+            Author = new CustomDbSet<Author>(connectionString);
+            Book = new CustomDbSet<Book>(connectionString);
+            Libraries = new CustomDbSet<Library>(connectionString);
+            Librarians = new CustomDbSet<Librarian>(connectionString);
+        }
+        public CustomDbSet<Person> People { get; set; }
+        public CustomDbSet<Author> Author { get; set; }
+        public CustomDbSet<Book> Book { get; set; }
+        public CustomDbSet<Library> Libraries { get; set; }
+        public CustomDbSet<Librarian> Librarians { get; set; }
+    }
+    public class Program
```

Comment:
all testing functionality should be in another project, and ORM should be class library

Reply:
Done